api/events/handlers.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
```

Comment:
I confess I opened my mind to other name conventions, I always used controller, but I already see handlers name instead of controller on others projects and I'm trying that :)